test/images/scripts/fedora-testing.setup

```diff
@@ -9,6 +9,8 @@ set -ex
 xfs_growfs /
 df -h /
 
+echo foobar | passwd --stdin root
```

Comment:
Sometimes this is used with virt-install, so that's why we have it fedora-testing (and fedora-24 until recently).